[11.0][IMP] mdc_crumbs: Added date to crum search + code cleanup

adriresu opened the pull request.

dalonsod commented on mdc_crumbs/__manifest__.py:
```diff
@@ -3,12 +3,12 @@
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "11.0.1.0.1",
+    "version": "11.0.1.1.1",
```
Wrong version number:
```suggestion
    "version": "11.0.1.1.0",
```

dalonsod reviewed (changes requested):
This commit is unclean. There are several changes that don't belong to the added functionality as described in the commit message.

Is it possible to split changes in two different commits, e.g. making a second commit with the other adjustments (changes in translations, views cleanup)?

adriresu committed:
[IMP] mdc_crumbs: Changed translations, code cleanup

adriresu committed:
[IMP] mdc_crumbs: Added date field to filter on crumbs views

adriresu commented:
Splitted commit into 2 commits, now:

[IMP] mdc_crumbs: Changed translations, code cleanup
[IMP] mdc_crumbs: Added date field to filter on crumbs views
